### ISSUE TITLE
Locality Confidence Bug

### ISSUE THREAD

**Issue by mczajkow** [mczajkow/mediamail#28]
I'm seeing the locality confidence not as a float error in this snippet of code:

```
        if localityConfidence is not None:
            lC = 0.0
            try:
                lC = float(locatlityConfidence)
            except:
                # Not a number?
                log.debug('Provided data has a locality confidence that is not a float. Ignoring.')
                return
            if lC < 0.0 or lc > 1.0:
                log.debug('Provided data has a locality confidence that is not between 0.0 and 1.0 inclusive. Ignoring.')
                return
            body['locality_confidence'] = lC
```

Address this issue and ensure locality and locality confidence are properly being set.

**Comment by mczajkow** [comment]
Locality is very important. Not seeing too many local tweets working - this is now more essential to address.

**Comment by mczajkow** [comment]
Hold off on this one for now. Confidence and Locality are being reconsidered / revamped in design.

**Comment by mczajkow** [comment]
When addressing #20 and making geo location points in the main ES index, I noticed a bug that likely is causing the problem here. Will test as part of resolving #20.

**Comment by mczajkow** [comment]
As suspected, this was fixed on #20 when trying to debug location as a geo_point. It is now addressed.
